examples/QueryDsl/MultiMatchQueryPage/0e118857b815b62118a30c042f079db1.asciidoc

```diff
@@ -0,0 +1,20 @@
+////
+IMPORTANT NOTE
+==============
+This file is generated from method Line259 in https://github.com/elastic/elasticsearch-net/tree/master/src/Examples/Examples/QueryDsl/MultiMatchQueryPage.cs#L277-L307.
+If you wish to submit a PR to change this example, please change the source method above
+and run dotnet run -- asciidoc in the ExamplesGenerator project directory.
+////
+[source, csharp]
+----
+var searchResponse = client.Search<object>(s => s
+    .Index(Nest.Indices.AllIndices)
+    .Query(q =>
+        q.MultiMatch(c => c
+            .Query("quick brown f")
+            .Type(TextQueryType.PhrasePrefix)
+            .Fields("subject, message")
```

Comment:
should we _discourage_ the comma separated string approach?

Reply:
It think its a matter of taste, but will move to arrays to be consistent.